src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/PInvokeMethodFixupNode.cs

```diff
@@ -91,8 +105,9 @@ public override int CompareToImpl(ISortableNode other, CompilerComparer comparer
         public readonly PInvokeModuleData ModuleData;
         public readonly string EntryPointName;
         public readonly CharSet CharSetMangling;
+        public readonly int? ObjectiveCMessageSendFunction;
```

Comment:
Do we need this as part of `PInvokeMethodData`? It raises unnecessary questions about whether the new state needs to be looked at in `Equals`.

Could we just compute it on demand in `PInvokeMethodFixupNode.GetData`?

Reply:
I removed this member.